app/models/settings.ts

```diff
@@ -6,6 +6,7 @@ export interface Server {
   username: string
   token: string
   salt: string
+  usePlainPassword: boolean // see ServerView.tsx for more information
```

Comment:
I'm realizing now that I haven't set up a pattern for migrating existing data in the DB when the schema changes, which isn't a huge deal when you're just adding a single boolean value like this, but I'd like all values to have their correct types and technically this would be `boolean | undefined` without migration, since it wouldn't exist until it is set for the first time.

That's a bit bigger change than just this feature though so I can add that in if that sounds alright with you.  Otherwise if you'd like to tackle that as well let me know!